js/base/Exchange.js

```diff
@@ -819,6 +819,108 @@ module.exports = class Exchange {
         throw new NotSupported (this.id + ' fetchPermissions() not supported yet')
     }
 
+    getInitialPermissions () {
+        const broadPropertyNames = {
+            'publicAPI': 0,
```

Comment:
Let's use `false` instead of `0`.

Reply:
I thought 'false' would have been a bit confusing, as they are not listed there to have any value actually, as later we use them just to find out if they are in 'array' (to use log-n approach, compared to 'in_array'). so, setting it to 'false' will just make a code few bits longer unnecessarily, and moreover might be confusing for someone on first sight (thinking their value is 'false'), while those values are of no meaning there, be it 0, 1 or whatever. 
lmk if you still think them to be `false`.